langmeta/langmeta/shared/src/main/scala/org/langmeta/io/Multipath.scala

```diff
@@ -63,6 +67,26 @@ sealed trait Multipath {
   def find(name: RelativePath): Option[URI] = {
     deep.find(_.name == name).map(_.uri)
   }
+
+  def visit(getVisitor: AbsolutePath => FileVisitor[Path]): Unit = {
```

Comment:
How about we make this an extension method? This will avoid the need to silence mima.

Reply:
This method deserves inclusion in Multipath IMO, and the mima warning is standard procedure when dealing with scala-specific features that are not first-class on the jvm like `sealed` and `private[pkg]`

Reply:
What exactly is the error saying? I think we need to make sure that our `Every program linkable against x.y.0 must be linkable against x.{y+1}.z` rule is not violated.

Reply:
The error says that if a client from external repo implements that interface then it will fail at runtime. Because the trait is `sealed` the language contract promises us that nobody extends `Multipath`

Reply:
In Scala.js they ignore mima reports all the time because of package private and sealed, one example https://github.com/scala-js/scala-js/blob/ec3c3bd685ff32f9fbbe4e19db085cb651babf8c/project/BinaryIncompatibilities.scala#L9

Reply:
Oh right, `sealed`. Makes sense then.